fixtures/test2-fixture.csv

```diff
@@ -0,0 +1,49 @@
+Term,Flight No,From,Scheduled,Estimated,Touchdown,Est Chocks,Act Chocks,Stand,Max pax,Act Pax,Trans pax

```

Comment:
We tend to use 'origin' rather than 'from'

Reply:
either lower or upper-case the second words in each field. we have a mixture here

Reply:
sorry, I copied this from LHR, I guess it's not the best example.